### PR TITLE
[UXE-3320] refactor: service unbinding treated as deletion in edge node

### PR DESCRIPTION
**Pull request by william-tome**
### What is the new behavior introduced by this PR?
Create `unbind-dialog` to handle unbinding confirmation instead of treating as deletion.
E.g.: edge node

* UI changes approved by design
* Text changes approved by tech writer

### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.
Screenshot 2024-07-30 at 12 40 22

### Does it have a link on Figma?
No
<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [x] Application responsiveness was tested to different screen sizes
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: src/views/EdgeNode/ListViewTabServices.vue [hunk 1]
```diff
@@ -87,10 +87,11 @@
...
-      type: 'delete',
-      title: 'service',
+      type: 'unbind',
+      title: 'edge service',
+      parent: 'edge node',
       icon: 'pi pi-trash',
-      service: deleteServicesWithDecorator
+      service: unbindServicesWithDecorator
     }
   ]
```
Comment (HerbertJulio) [inline src/views/EdgeNode/ListViewTabServices.vue:96]: Recomendo utilizar o `type: 'dialog'`

File: src/templates/list-table-block/index.vue [hunk 2]
```diff
@@ -374,6 +375,23 @@
...
+                    selectedItemData: rowData,
+                    deleteDialogVisible: true,
+                    unbindingService: action.service,
+                    rerender: Math.random()
+                  },
+                  onClose: (opt) => opt.data.updated && reload()
+                }
+                openDialog(UnbindDialog, bodyUnbind)
+              }
+              break
```
Comment (HerbertJulio) [inline src/templates/list-table-block/index.vue:394]: Recomendo utilizar o case de Dialog

File: src/templates/list-table-block/dialog/unbind-dialog.vue [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+  watch(
+    () => confirmation.value,
+    (value) => {
+      if (value) {
+        canUnbind.value = true
+      }
+    },
+    { deep: true }
+  )
```
Comment (HerbertJulio) [inline src/templates/list-table-block/dialog/unbind-dialog.vue:183]: Acredito que não haja necessidade de utilizar watch, recomendo deixar btn habilitado é exibir um error no campo como obrigatório.
Se o padrão de delete for disabled, então adicionar o confirmation.value no isDisabled.

File: src/templates/list-table-block/dialog/unbind-dialog.vue [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+  }
+
+  const cancelDialog = () => {
+    resetForm()
+    dialogRef.value.close({ updated: false })
+  }
+
+  const getLoadingIcon = computed(() => {
+    return loading.value ? 'pi pi-spin pi-spinner' : ''
+  })
```
Comment (HerbertJulio) [inline src/templates/list-table-block/dialog/unbind-dialog.vue:158]: Recomendo utilizar a própria prop de `loading` do PrimeButton.

File: src/templates/list-table-block/dialog/unbind-dialog.vue [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+        label="Cancel"
+        @click="cancelDialog()"
+        data-testid="unbind-dialog__footer__cancel-button"
+      />
+      <PrimeButton
+        severity="danger"
+        label="Unbind"
+        icon-pos="right"
+        @click="removeItem()"
+        :icon="getLoadingIcon"
```
Comment (HerbertJulio) [inline src/templates/list-table-block/dialog/unbind-dialog.vue:85]: Recomendo utilizar a própria prop de `loading` do PrimeButton.

Comment (william-tome) [reply src/templates/list-table-block/dialog/unbind-dialog.vue:85]: Resolvido

File: src/templates/list-table-block/index.vue (hunk 2 above)
Comment (william-tome) [reply src/templates/list-table-block/index.vue:394]: Valeu pela recomendação. Alterado aqui.

File: src/templates/list-table-block/dialog/unbind-dialog.vue (hunk 3 above)
Comment (william-tome) [reply src/templates/list-table-block/dialog/unbind-dialog.vue:183]: Resolvido

File: src/views/EdgeNode/ListViewTabServices.vue (hunk 1 above)
Comment (william-tome) [reply src/views/EdgeNode/ListViewTabServices.vue]: Feito

**Review by lucasmendes21 — APPROVED** [review]

**Review by pauloSF0 — APPROVED** [review]

**Pull request merged**